site/static/admin/config.yml

```diff
@@ -48,7 +49,7 @@ collections:
           pattern: ['.{144,}', "￼Must be a valid SharePoint link"],
           hint: "Always open SharePoint links on Internet Explorer",
           required: true }
-      - { name: date, label: Publish Date, widget: dateTimeWithNow,
+      - { name: date, label: Publish Date, widget: datetime,
```

Comment:
This change is not part of this work, it is part of ONS-795.

Reply:
true - but it was necessary to remove it here, because that was what got the build to fail